package.json

```diff
@@ -148,8 +149,9 @@
     ]
   },
   "lint-staged": {
-    "src/**/*.{ts,tsx}": [
+    "src/**/**/*.{ts,tsx}": [
       "eslint --fix",
+      "yarn prettier --write",
```

Comment:
Same here, not sure of `yarn` call need.

Reply:
```suggestion
      "prettier --write",
```